lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
I personally never used bare repos. Can they actually be used in deployments to production?